packages/shared/styles/components/SfTabs.scss

```diff
@@ -86,6 +85,7 @@ $tabs__content-padding-y: 3 * $spacer-big !default;
     }
   }
   &__chevron {
+    margin-top: calc(calc(calc(#{$tabs__title-font-size} * #{$tabs-line-height}) - 20px) / 2);
```

Comment:
You can do this without `calc` -`calc` is very harmful for layout frame rates, it's better to omit it when possible